receiver/dockerstatsreceiver/metadata.yaml

```diff
@@ -628,3 +628,19 @@ metrics:
       aggregation: cumulative
     attributes:
       - interface
+
+  # Pids
+  container.pids.count:
```

Comment:
Related to my comment about the semantic convention: also consider the naming of the metric and try and somewhat align it to the semantic convention. Do you think `container.processes.count` is better for alignment? I'll let you make that call, but it's something to consider.

Reply:
Apparently `processes` is more widely used and is more aligned with the semantic convention, but taking into account that the value is named `pids` in [cgroups](https://www.kernel.org/doc/Documentation/cgroup-v1/pids.txt) and docker API does not change it, maybe keeping `pids` here makes new metrics easier to identify and understand. WDYT?

Reply:
Yeah, I tend to agree. It's reasonable to keep it as `pids` here.